lms/djangoapps/instructor_analytics/basic.py

```diff
@@ -222,6 +224,15 @@ def enrolled_students_features(course_key, features):
     if include_team_column:
         students = students.prefetch_related('teams')
 
+    def extract_attr(student, feature):
```

Comment:
This is because 'country' is not a string: it's an instance of CountryField. When we try to serialize `student_dict` as JSON, serialization will fail for the values that do not have an acceptable type (int, str, list, etc.). That piece of code with DjangoJSONEncoder was inspired by the JSON-conversion code from django models.